src/test/java/teammates/test/cases/ui/InstructorFeedbackAddActionTest.java

```diff
@@ -37,149 +39,172 @@ public void testExecuteAndPostProcess() throws Exception{
         //TODO make sure IFAA does assertNotNull for required parameters then uncomment
         //verifyAssumptionFailure(Const.ParamsNames.COURSE_ID, instructor1ofCourse1.courseId);
         
+        
         ______TS("Typical case");
         
-        String[] params =
-                createParamsCombinationForFeedbackSession(
-                        instructor1ofCourse1.courseId, "ifaat tca fs", 0);
+        String[] params = createParamsCombinationForFeedbackSession(
+                instructor1ofCourse1.courseId, "ifaat tca fs", 0);
```

Comment:
How much should I indent it by? 8 spaces starting from the parent element's first column?
````
createParamsCo...
        instructor...
````